src/env.ts

```diff
@@ -98,6 +98,10 @@ const CORECLR_PROFILER = "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}";
 const CORECLR_PROFILER_PATH = "/opt/datadog/Datadog.Trace.ClrProfiler.Native.so";
 const DD_DOTNET_TRACER_HOME = "/opt/datadog";
 
+// Java tracer env variables
+const JAVA_TOOL_OPTIONS_VAR = "JAVA_TOOL_OPTIONS";
+const JAVA_TOOL_OPTIONS = '-javaagent:"/opt/java/lib/dd-java-agent.jar" -XX:+TieredCompilation -XX:TieredStopAtLevel=1';
```

Comment:
I've noticed some error logs from the agent, which seems to occur because of the missing env. config setting `DD_JMXFETCH_ENABLED: false` - see [DD configure-environment-variables](https://docs.datadoghq.com/serverless/installation/java/?tab=gradle#configure-environment-variables)...

<img width="1041" alt="image" src="https://user-images.githubusercontent.com/81381/161949114-18d2b035-a5ea-4521-9581-c129deada7d7.png">

- [ ] include `DD_JMXFETCH_ENABLED: false` environment config